app/controllers/person_controller.rb

```diff
@@ -0,0 +1,18 @@
+class PersonController < ApplicationController
+  def index
+    @person = Person.all
+    render :index
+  end
+
+  def show
+    @id = params[:id]
+    @person = Person.find(@id)
```

Comment:
Your show view is only using the `@person` instance variable so you should use a local variable for the `id` rather than an instance variable.